gui/src/app/tinystan/index.ts

```diff
@@ -280,17 +305,35 @@ export default class StanModel {
       );
 
       // copy out parameters of interest
-      const draws: number[][] = Array.from({ length: n_params }, () => []);
-      for (let i = 0; i < n_draws; i++) {
-        for (let j = 0; j < n_params; j++) {
-          const elm = out_buffer[i * n_params + j];
-          draws[j][i] = elm;
-        }
-      }
+      const draws: number[][] = Array.from({ length: n_params }, (_, i) =>
+        Array.from({ length: n_draws }, (_, j) => out_buffer[i + n_params * j])
```

Comment:
This went from "multiply by `i` and add `j`" to the other way around. Looks like `i` is still the outer index and `j` the inner--what's up? Did the memory layout change?

Reply:
The memory layout didn't change, just the previous way of writing it was a bit opaque -- before, `i` was from 0 to `num_draws`, and `j` went from 0 to `num_params`, which is why the assign in to draws was `[j][i]` -- flipped from how you would expect.

In addition to being a bit more compact, this style of just directly using `Array.from` forced me to actually write my indexes in the order they should be